src/scaict_uwu/libs/language/language_tag_factory.py

```
@@ -0,0 +1,55 @@
+    The LanguageTagFactory class deals with LanguageTag object creations.
+    """
+
+    _tags: dict[str, LanguageTag] = {}
+    """
+    _tags (dict): The LanguageTag objects.
+    """
+
+    def __init__(self) -> None:
+        pass
+
+    def get_tag(self, tag) -> LanguageTag:
```

Ruff (0.8.0)

Missing type annotation for function argument `tag`

(ANN001)